src/test/scala/eventstore/SoftDeleteStreamITest.scala

```diff
@@ -112,6 +116,7 @@ class SoftDeleteStreamITest extends TestConnection {
     def appendEventToRecreate(expVer: ExpectedVersion = ExpectedVersion.Any): EventData = {
       val event = newEventData
       write(expVer, event)
+      Thread.sleep(100) // Give ES time - fix this
```

Comment:
could you explain this?

Reply:
Just me trying fix tests due to slow ES/CI. Same as above